airbyte-webapp/src/views/Connector/ConnectorForm/components/FrequentlyUsedConnectors/FrequentlyUsedConnectorsCard.tsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import { useIntl } from "react-intl";
+
+import { ConnectorCard } from "components";
+import { SlickSlider } from "components/ui/SlickSlider";
+
+import { ConnectorCard as ConnectorCardType } from "../../types";
+import styles from "./FrequentlyUsedConnectorsCard.module.scss";
+
+export interface FrequentlyUsedConnectorsCardProps {
+  connectors: ConnectorCardType[];
+  connectorType: "source" | "destination";
+  onConnectorSelect: (id: string, connectorName: string) => void;
+}
+
+export const FrequentlyUsedConnectorsCard: React.FC<FrequentlyUsedConnectorsCardProps> = ({
+  connectors,
+  onConnectorSelect,
+  connectorType,
+}) => {
+  const { formatMessage } = useIntl();
+
+  if (!connectors?.length) {
```

Comment:
very nitpicky 😁 take it or leave it
```suggestion
  if (connectors.length === 0) {
```

Reply:
I like it more!